content/docs/guides/crosswalk/aws/vpc.md

```diff
@@ -474,6 +474,84 @@ sg.createEgressRule("outbound-access", {
 For additional details about configuring security group rules, See the
 [Security Groups for Your VPC](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html) documentation.
 
+## How to use your VPC, Security Group, and EC2 instance
+
+```typescript
+import * as awsx from "@pulumi/awsx";
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+
+// Allocate a new VPC using any of the above techniques.
+const vpc = new awsx.ec2.Vpc("custom", { /*...*/ });
+
+// Allocate a security group using the above techniques:
+const sg = new awsx.ec2.SecurityGroup("webserver-sg", { vpc });
+
+// t2.micro is available in the AWS free tier
+const size = "t2.micro";
+
+// Get the most recent Amazon linux ami:
+const ami = pulumi.output(aws.getAmi({
+    filters: [{
+        name: "name",
+        values: ["amzn-ami-hvm-*"],
+    }],
+    owners: ["137112412989"], // This owner ID is Amazon
+    mostRecent: true,
+}));
+
+const server = new aws.ec2.Instance("webserver-www", {
+    instanceType: size,
+    vpcSecurityGroupIds: [ sg.id ], // reference the security group resource above
+    subnetId: pulumi.output(vpc.publicSubnetIds)[0],  // reference the public subnet from the custom vpc above
+    ami: ami.id,
+});
+```
+
+This example launches an ec2 instance in the vpc and security group that was created above.
```

Comment:
```suggestion
If we run `pulumi up`, the `aws.ec2.Instance` will be provisioned using the _first_ public subnet from the `awsx.ec2.Vpc` component and the security group provisioned with the `awsx.ec2.SecurityGroup` component:
```

Reply:
Updated the content to reflect the above